components/aws/sagemaker/tests/unit_tests/tests/test_hpo.py

```diff
@@ -59,7 +59,7 @@ def test_spot_local_path(self):
     self.assertEqual(response['TrainingJobDefinition']['CheckpointConfig']['S3Uri'], 's3://fake-uri/')
     self.assertEqual(response['TrainingJobDefinition']['CheckpointConfig']['LocalPath'], 'local-path')
 
-  def test_empty_string(self):
+  def check_empty_string(self):
```

Comment:
I still don't feel this really describes what the test is trying to achieve. I think this test name is used in every file, now that I look into it. So perhaps just for consistency we should just revert it back. Sorry about that.